Build/Laptop/EMail.cc

```diff
@@ -1916,7 +1917,7 @@ static void HandleIMPCharProfileResultsMessage(void)
 		// have to place players name into string for first record
 		if (i == 0)
 		{
-			wchar_t zTemp[512];
+			wchar_t zTemp[NAME_LENGTH];
```

Comment:
If it contains 30 chars, yes. But if that happens, we'll at least see it instead of it only overflowing within the structure. We can adjust NAME_LENGTH easily whenever this should happen